file.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello Microverse project</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [ ] [OPTIONAL] Kindly add a `class` attribute to the ` h1 ` tag and update your `CSS `to the `class` attribute according to the project requirement.